lib/pages/home/widgets/dashboard_header.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:flutter/material.dart';
+import 'package:school_erp/pages/profile/profile_page.dart';
+import '../../../constants/text_constants.dart';
+import '../../../features/auth/auth_repository/schemas/user.dart';
+import '../../../theme/text_styles.dart';
+import '../../EnterExitRoute.dart';
```

Comment:
i think prefer to use abs import for explicitness / clearness. Especially for the case of going 3 levels up